### PR TITLE
Del schema

### PR DESCRIPTION
**Pull request by hinthornw**
i really want this

### CODE REVIEW

File: libs/core/langchain_core/tools.py [hunk 1]
```diff
@@ -290,7 +290,7 @@ def create_schema_from_function(
     # Pydantic adds placeholder virtual fields we need to strip
     valid_properties = _get_filtered_args(inferred_model, func, filter_args=filter_args)
     return _create_subset_model(
-        f"{model_name}Schema",
```
Comment (RafaelXokito) [inline libs/core/langchain_core/tools.py:293]: You need to take into account the users that might rely on this name.

**Review by RafaelXokito — CHANGES_REQUESTED** [review]

File: libs/core/langchain_core/tools.py (hunk 1 above)
Comment (eyurtsev) [reply libs/core/langchain_core/tools.py:293]: @hinthornw we could try and make this change as part of 0.3.

@RafaelXokito are you relying on this feature?

Comment (RafaelXokito) [reply libs/core/langchain_core/tools.py:293]: @eyurtsev no I don't, thank you for asking.
I thought it would be good idea to  consider that scenario.

**Comment by hinthornw** [comment]
Ya not going to land this in 0.2

**Comment by eyurtsev** [comment]
change was made in v0.3

**Pull request closed without merging**